### PR TITLE
Feat/convert task

### PR DESCRIPTION
**Pull request by MaxPostema**

### CODE REVIEW

**Comment by mswertz** [comment]
Hi, there in https://github.com/molgenis/molgenis-emx2/pull/1520 I had to already migrate task and subtask.
So you can either take those or simply skip them.

**Comment by MaxPostema** [comment]
> Hi, there in #1537 I had to already migrate task and subtask. So you can either take those or simply skip them.

I suggest using this variant of the Task component as we removed a 'memory leak' fixed it cluttering up the error log and cleaned up some of the used code style.

**Review by connoratrug — APPROVED** [review]

File: apps/molgenis-components/src/components/task/Task.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+</script>
+
+<docs>
+<template>
+  <div>
+    <demo-item>
+      <Task taskId="6956bf6d-3f78-4798-873f-dd5382ac0e24"/>
```
Comment (mswertz) [inline apps/molgenis-components/src/components/task/Task.vue:79]: I think we should add a stringinput where you can enter the taskID.
What you also can do is to only demo as part of taskList

**Review by mswertz — DISMISSED** [review]

**Comment by mswertz** [comment]
PR looks good, I think only task demo should be parameterized (with stringinput to provide a taskid) otherwise it will never really work.

**Pull request merged**